### PR TITLE
feat(Kickstarter): add presence

### LINKED ISSUE #5303: Kickstarter | kickstarter.com
### Discussed in https://github.com/PreMiD/Presences/discussions/4605

<div type='discussions-op-text'>

<sup>Originally posted by **csantos987** May 24, 2021</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 

**What's the name of the service?** 

https://kickstarter.com | Kickstarter

**What should the Presence display?** 

Line 1: whether a user's browsing the home page, search, or a campaign page
Line 2: if in a search query, show search terms. If on a campaign page, show title of campaign page

**If possible, please provide a logo for the service (512x512 minimum)** 

https://play-lh.googleusercontent.com/r4wBopBrIEeH8ICromEm1mDuzawUquXzZzYmeG22nqy_M0_KyRJKNn2v4gjfcsiOZZ0</div>

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

- Resolves #5303 by creating the requested presence.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

kickstarter_proof_5

kickstarter_proof_4

kickstarter_proof_3

kickstarter_proof_2

kickstarter_proof_1
</details>

### CODE REVIEW

File: websites/K/Kickstarter/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		"id": "240521747852558347"
+	},
+	"service": "Kickstarter",
+	"description": {
+		"en": "Kickstarter exists to help bring creative projects to life. A home for film, music, art, theater, games, comics, design, photography, and more.",
+        "de": "Kickstarter existiert, um kreativen Projekten zur Verwirklichung zu verhelfen. Hier findest du Filme, Musik, Kunst, Theater, Spiele, Comics, Design, Fotografie und vieles mehr.",
+		"fr": "Kickstarter existe pour faire vivre les projets créatifs. Un site dédié à la musique, à l'art, au cinéma, au théâtre, à la BD, au design, à la photographie et aux jeux, entre autres.",
+        "es": "Kickstarter existe para ayudar a hacer realidad proyectos creativos. Un lugar para el cine, la música, el arte, el teatro, los juegos, las historietas, el diseño, la fotografía y más.",
+        "it": "Kickstarter esiste per aiutare a dare vita ai progetti creativi. Kickstarter accoglie tanti progetti: film, musica, arte, teatro, videogiochi, fumetti, grafica, fotografia e tanto altro.",
+        "ja_JP": "Kickstarterは、映画、音楽、アート、シアター、ゲーム、コミック、デザイン、写真など、多種多様なクリエイティブプロジェクトを実現させるための場です。"
```
Comment (EncryptedDev) [inline websites/K/Kickstarter/dist/metadata.json:14]: Formatting has seemed to messed itself up here; it is a combination of tabs and spaces

File: websites/K/Kickstarter/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	},
+	"url": "www.kickstarter.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/evUBiFh.png",
+	"thumbnail": "https://i.imgur.com/Kv6FL16.png",
```
Comment (EncryptedDev) [inline websites/K/Kickstarter/dist/metadata.json:19]: This is very elongated which will most likely cause a issue when displaying

File: websites/K/Kickstarter/presence.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({ clientId: "1014441192106229790" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	slideshow = presence.createSlideshow(),
+	staticsTabsProject: { [name: string]: string } = {
+		description: "Reading project description",
+		faqs: "Reading project FAQ",
+		posts: "Reading project news",
+		comments: "Reading project comments",
+		community: "Viewing project contributors",
+		pledge: "Contributing to the project",
```
Comment (EncryptedDev) [inline websites/K/Kickstarter/presence.ts:10]: You can use Title Case here

**Review by EncryptedDev — COMMENTED** [review]
Hiya! Take a look at my review below please 🚀

File: websites/K/Kickstarter/dist/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/K/Kickstarter/dist/metadata.json:14]: I'll see to resolve that!

File: websites/K/Kickstarter/presence.ts (hunk 3 above)
Comment (RisingSunLight42) [reply websites/K/Kickstarter/presence.ts]: Resolved in 1fdbed86bf8344a15b1a6430d2b4aa244e35ee9f

File: websites/K/Kickstarter/dist/metadata.json (hunk 2 above)
Comment (RisingSunLight42) [reply websites/K/Kickstarter/dist/metadata.json]: Resolved in 1fdbed86bf8344a15b1a6430d2b4aa244e35ee9f

File: websites/K/Kickstarter/dist/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/K/Kickstarter/dist/metadata.json]: Resolved in 1fdbed86bf8344a15b1a6430d2b4aa244e35ee9f

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**